next.config.js

```diff
@@ -56,6 +56,7 @@ const nextConfig = {
       'api': path.join(__dirname, 'api'),
       'main': path.join(__dirname, 'main'),
       'config': path.join(__dirname, 'config.js'),
+      'store': path.join(__dirname, 'store'),
```

Comment:
I would maybe call it `model` but it can be changed (or not) later, I hope. Do you have any preference?

Reply:
I don't mind either. I am just a big fan of using the same naming across all our projects.

Reply:
Sure, it can be renamed everywhere for sure but we should decide what we really have.